auth_signup_verify_email/controllers/main.py

```diff
@@ -34,7 +34,7 @@ def passwordless_signup(self, values):
             values["email"] = values.get("login")
 
         # Remove password
-        values["password"] = False
+        values["password"] = ""
```

Comment:
But I have searched through the code and cannot see where it breaks, could you point me to it please? The field is not required after all...